python/lsst/ts/mtrotator/mock_controller.py

```diff
@@ -320,6 +320,9 @@ async def update_telemetry(self, curr_tai):
             self.telemetry.current_pos = curr_pos
             self.telemetry.current_vel_ch_a_fb = curr_segment.velocity
             self.telemetry.current_vel_ch_b_fb = curr_segment.velocity
+            self.telemetry.motor_current[:] = [motor_current] * 2
+            self.telemetry.bus_voltage = 20  # arbitrary
+            # These fields seem to duplicate telemetry.motor_current?
             self.telemetry.motor_current_axis_a = motor_current
             self.telemetry.motor_current_axis_b = motor_current
```

Comment:
What is the difference among the `self.telemetry.motor_current[:]`, `self.telemetry.motor_current_axis_a`, and `self.telemetry.motor_current_axis_b`?

Reply:
I reply this by myself. They are the relationship of outputs and inputs:
1. https://github.com/lsst-ts/ts_rotator_controller/blob/develop/include/interface/telemetryStructure.h#L101-L102
2. https://github.com/lsst-ts/ts_rotator_controller/blob/develop/include/interface/telemetryStructure.h#L183-L192

Thanks!

Reply:
I was asking you that question! But you answered it on Slack. You can see how these values map to the telemetry struct by looking at the structs.py file. It turns out motor_current is measured motor current, whereas the motor_current_axis_a/b fields are demand motor current.